### PR TITLE
CLDR-14138 CLDRModify -fp

### PR DESCRIPTION
**Pull request by srl295**
CLDR-14138

- Also, no longer generate empty /diff/log.txt files.
This log was no longer used, so it was removed from the
code.

Modifications of:
- CLDR-14055 Add annotations for new locales (#627)
- CLDR-10102 Seed latin (#578)

##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-_____
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by srl295** [comment]
- added derived annotations

File: common/annotations/my.xml [hunk 1]
```diff
@@ -1381,7 +1381,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<annotation cp="🐡" type="tts">ငါးပူဖောင်း</annotation>
 		<annotation cp="🦈">ငါး | ငါးမန်း</annotation>
 		<annotation cp="🦈" type="tts">ငါးမန်း</annotation>
-		<annotation cp="🐙">ဘဝှဲ ရဘဝှေဲ ဘဝှဲ ဘဝွဲ ဘဝြဲ ဘဝဲ</annotation>
+		<annotation cp="🐙">ဘဝှဲ ရဘဝှေဲ ဘဝှဲ ဘဝှဲ ဘဝွဲ ဘဝဲ</annotation>
```
Comment (macchiati) [inline common/annotations/my.xml:1384]: I think you have to revert this change in my.xml. (There was an earlier problem with the Zwagyi "fixer" in DAIP)

File: common/annotationsDerived/af.xml [hunk 2]
```diff
@@ -462,6 +462,86 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="👱🏾" type="tts">persoon met blonde hare: medium-donker velkleur</annotation>
 		<annotation cp="👱🏿">blond | donker velkleur | hare | persoon met blonde hare</annotation>
 		<annotation cp="👱🏿" type="tts">persoon met blonde hare: donker velkleur</annotation>
+		<annotation cp="🧑🏻‍❤‍💋‍🧑🏼">ligte velkleur | medium-ligte velkleur | paartjie | romanse | soen | volwassene</annotation>
```
Comment (macchiati) [inline common/annotationsDerived/af.xml:465]: I think these new ones happened as a result of the updates for Emoji 13.1. My fault for not rerunning; glad you did this.

**Review by macchiati — CHANGES_REQUESTED** [review]
Just one fix necessary; reverting my.xml

**Comment by srl295** [comment]
Understood, will do tomorrow.

Any more steps needed? -fQ? Transformed locales?

**Comment by macchiati** [comment]
None of the new ones are transformed, so that's fine. But, doooh:


   - I should have said you need -fQ also, but just for the annotations
   directory.
   - You don't need to do -fp afterwards, but you do need to do the
   derivedAnnotations.


Mark

**Comment by srl295** [comment]
@pedberg-icu this also modifies la.xml and I think passes tests.

**Comment by srl295** [comment]
> But, doooh: 

From a process improvement perspective, we'll call this a success :)

**Comment by macchiati** [comment]
Yup!

**Comment by srl295** [comment]
@macchiati seems `-fQ` does not work for seed/annotations (`Annotations.LOCALES` is built explicitly from `common/annotations`). Is this a bug, or just skip -fQ for seed?

**Comment by srl295** [comment]
ok, it seems there aren't any derivedAnnotations for seed, for similar reasons…

**Comment by srl295** [comment]
whoops. forgot to exclude `my.xml` doing this now:

- removing my.xml from the commit
- re-running `modify -fQ` but not `modify -fp`
- re-running `GenerateDerivedAnnotations`


And for the record, I did similar for `seed/main/la.xml` - removed it, re-ran `modify -fp`.

File: common/annotations/my.xml (hunk 1 above)
Comment (srl295) [reply common/annotations/my.xml]: fixed in c7b1ea206bbfc2ef399019aafd4350858803a7cc

**Comment by srl295** [comment]
@macchiati 
- `common/main/my.xml` did not seem to be affected by -fp
- reverting `common/annotations/my.xml` as above did not seem to produce a new derived annotation.

**Comment by srl295** [comment]
> @macchiati
> 
> * `common/main/my.xml` did not seem to be affected by -fp
> * reverting `common/annotations/my.xml` as above did not seem to produce a new derived annotation.

@macchiati think we are good to go. https://github.com/unicode-org/cldr/pull/691/files#diff-a874a048457c728fe7277edcbffdb107 does not show any scrambled Myanmar anymore.

**Review by macchiati — APPROVED** [review]
